velox/type/Timestamp.cpp

```diff
@@ -94,9 +94,11 @@ Timestamp::toTimePointMs(bool allowOverflow) const {
   return tp;
 }
 
-void Timestamp::toTimezone(const tz::TimeZone& zone) {
+void Timestamp::toTimezone(const tz::TimeZone& zone, bool allowOverflow) {
   try {
-    seconds_ = zone.to_local(std::chrono::seconds(seconds_)).count();
+    auto tp = toTimePointMs(allowOverflow);
```

Comment:
why do we need to create a time point here just to discard it later? The same validateRange() method will be called inside to_local()